src/cli/export.rs

```diff
@@ -7,8 +7,11 @@ use surrealdb::Error as SurrealError;
 
 #[tokio::main]
 pub async fn init(matches: &clap::ArgMatches) -> Result<(), Error> {
-	// Set the default logging level
-	crate::cli::log::init(1);
+	// Initialize o11y
+	crate::o11y::builder().with_log_level(
+		matches.get_one::<String>("log").unwrap()
+	).init();
+
```

Comment:
The `--log` argument isn't customisable on the `surreal export` command. It was set to `error` only.

Reply:
Here we only want `error`, not `warn`, `info`, `debug`, `trace`, which is why it was hardcoded to `error`.

Reply:
Asa side note, there currently is no `--log` argument for the `surreal export` command, so this wouldn't work anyway. If a user specifies `--log` on the `export` command, it will return an error saying that the argument is not recognised here.